### ISSUE TITLE
Suggested setup change

### ISSUE THREAD

**Issue by madsem** [aaemnnosttv/valet-dashboard#4]
Hey, neat little dashboard, I love it :)

Small suggestion, in your readme you write to link the dashboard.
According to this issue: #https://github.com/laravel/valet/issues/748

There is an undocumented settings key for the `~/.config/valet/config.json` file:
```json
"default": "/Users/yourusername/Sites/valet-dashboard",
```

Simply adding this makes the dashboard the default, so it is served under http://localhost

And also, maybe it would be a good idea to exclude the `valet-dashboard` directory in the list of sites?

Thoughts?

**Comment by madsem** [comment]
Okay I went ahead and made a pull request, maybe you like the changes :)

**Comment by aaemnnosttv** [comment]
Oh hi 👋 😄 
Thanks for opening this issue first, although I apologize, I saw your PR first and already responded there.

I think there's some great ideas there, and I appreciate your contribution. The default site thing is really cool. I won't go through everything again here but there's a fairly comprehensive response to all of your suggested changes in the PR (#5 ).

I'm open to accepting some PRs for changes but this one is a bit more than I'd like. See the PR for details, and I look forward to merging your contributions 😄 

Thanks again!

**Comment by madsem** [comment]
Alright, sounds good, happy you like some of the ideas.
When I need some time to relax this weekend (going to be a work weekend), I'll re-do everything with separate PR's.

A few questions:

- Would you be open to include a link to the info.php (Maybe somewhere top right corner, styled nicely maybe with fontawesome info icon).

- I just had the idea to maybe add a config.php, with the following settings:
  * Public paths & folders (array). Those are checked for existence for each site, that way you can enter paths for various frameworks / CMS etc. If no path exists, it is considered a "Package/Library" and listed in a new column titled just that :)
  * Option to list directory names you wish to exclude from being listed entirely (array)

**Comment by aaemnnosttv** [comment]
> Would you be open to include a link to the info.php

I'd prefer not to. Like I said, it's super easy to add it as a site if you'd like it handy.

> I just had the idea to maybe add a config.php
> * Public paths & folders (array). Those are checked for existence for each site, that way you can enter paths for various frameworks / CMS etc. If no path exists, it is considered a "Package/Library" and listed in a new column titled just that :)

This sounds more like an issue of how you're using Valet. Are you mixing directories for libraries and packages in a parked directory? If so, I would move your sites to a dedicated directory that is parked just for sites. Also not sure why you would have a package or library in the dashboard if it weren't openable as a site. If you wanted these in a separate column, you can just create a dedicated directory for these and park it! Remember, each parked directory becomes a new column 😄 

> * Option to list directory names you wish to exclude from being listed entirely (array)

This sounds similar to the other one. At first glance, it doesn't seem necessary if you're structuring/organizing your parked directories with sites only.

Am I understanding correctly? The project is intended to be minimal, so if it can be solved without additional files, conventions, etc, I'd prefer to avoid adding things for those cases.

**Comment by madsem** [comment]
Makes sense, okay :)
I just personally like to have all my code in one directory, not scattered across several. So I thought this could be useful for people like me!

**Comment by aaemnnosttv** [comment]
You can always use your own version! No reason why you can't customize it to your own needs and preferences 😄 

One other way you could "weed out" the libraries and other directories would be to unpark the directory, and use `valet link` for the individual sites.

Thanks for your suggestions. Looking forward to those PRs 😄

**Comment by madsem** [comment]
Yeah thanks, I think I'll roll my own and make those PRs separate :)
Well, you could for example list the packages/libraries in another column and then have them open your IDE, in case of PhpStorm for example `phpstorm://open?url=file:///`.

Similar to Laravels error view
